Plane.py

```diff
@@ -0,0 +1,47 @@
+import pygame
```

Comment:
Please complete documentation for all modules

Reply:
Done